bool calib_cmd

---

bool calib_status